packages/bpk-component-swap-button/src/BpkSwapButton.module.scss

```diff
@@ -52,6 +50,10 @@
       background-color: $bpk-surface-highlight-day;
     }
 
+    span {
+      line-height: $bpk-line-height-xs;
+    }
```

Comment:
Ideally would have liked to use the `withAlignment` HOC here but that was adding a margin top of 0.25rem which made shifted the svg slightly and stoped it rotating from the center